packages/govern-console/src/containers/DAOSettings/DAOSettings.tsx

```diff
@@ -243,294 +242,65 @@ const DaoSettings: React.FC = () => {
     });
   };
 
+  /**
+   * Render
+   */
   return (
-    <PageContent card={<SettingsCard />}>
```

Comment:
Why is this removed (and not replaced)? The `SettingsCard` component is still part of the design on Figma.

Reply:
The `SettingsCard` is removed in the new designs... We're only showing the `HelpComponent`